docs/deployment.rst

```diff
@@ -0,0 +1,144 @@
+.. include:: global.rst.inc
+.. _deployment:
+
+Deployment
+==========
+
+This chapter will give an example how to setup a borg storage server for multiple
+clients.
+
+Machines
+--------
+
+There are multiple machines used in this chapter and will further be aliased by their
+respective fully qualified domain name (fqdn).
+
+* The backup server: `backup01.srv.local`
+* The clients:
+  * John Doe's desktop: `johndoe.clnt.local`
+  * Webserver 01: `web01.srv.local`
+  * Application server 01: `app01.srv.local`
+
+User and group
+--------------
+
+The storage server needs to have only one UNIX user for all the clients.
+Recommended user and group and additional settings:
+
+* User: `backup`
+* Group: `backup`
+* Shell: e.g `/bin/bash` which is capable to run the `borg serve` command
+
+The clients probably will initiate a backup from the root user to catch all
+users, groups and permissions (e.g when backupping `/home`).
```

Comment:
In context with the paragraph above, I think it's a strange way of phrasing. I would just say

>* Shell: `/bin/bash` (or other capable to run the `borg serve` command)